packages/grid/_modules_/grid/hooks/root/useApiMethod.ts

```diff
@@ -21,7 +21,13 @@ export function useApiMethod(apiRef: ApiRef, apiMethods: Partial<GridApi>, apiNa
       if (!apiRef.current.hasOwnProperty(methodName)) {
         logger.debug(`Adding ${apiName}.${methodName} to apiRef`);
         apiRef.current[methodName] = (...args) => {
-          return apiMethodsRef.current[methodName](...args);
+          const start = new Date().getTime();
+          const result = apiMethodsRef.current[methodName](...args);
+          const end = new Date().getTime();
+          if(end - start > 10) {
+            console.log(`${methodName} took ${end - start}`);
+          }
+          return result;
```

Comment:
Did you intend to merge the performance diff or was it to debug?

Reply:
No it's just for debugging